mintpy/prep_fringe.py

```diff
@@ -329,6 +438,15 @@ def main(iargs=None):
     # metadata
     meta = prepare_metadata(inps.metaFile, geom_src_dir, box=src_box)
 
+    # optionally apply user multilooking
+    meta['AZIMUTH_PIXEL_SIZE'] = str(float(meta['AZIMUTH_PIXEL_SIZE']) * inps.lks_y)
+    meta['RANGE_PIXEL_SIZE'] = str(float(meta['RANGE_PIXEL_SIZE']) * inps.lks_x)
+    meta['azimuthPixelSize'] = meta['AZIMUTH_PIXEL_SIZE']
+    meta['rangePixelSize']   = meta['RANGE_PIXEL_SIZE']
```

Comment:
So far, all the mintpy functions have been only maintaining consistency for metadata listed here (https://mintpy.readthedocs.io/en/latest/api/attributes/), I would recommend removing these two lines for consistency, even though they are reasonable.

Reply:
sure, was curious as to why there was a duplicate